xml/System.Text.RegularExpressions/RegexRunner.xml

```diff
@@ -1535,8 +1535,8 @@
         <Parameter Name="text" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="0" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="text">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="text">The span of characters to scan for a pattern match.</param>
+        <summary>Used by a <see cref="T:System.Text.RegularExpressions.Regex" /> object generated by the <see cref="Overload:System.Text.RegularExpressions.Regex.CompileToAssembly" /> method.</summary>
```

Comment:
It's used by code generated by the source generator, not by Regex.CompileToAssembly.

Reply:
Nice, thanks! I'll revert this to To be added and will let the porting tool put the correct full names.